app/java/net/openid/appauthdemo/IdentityProvider.java

```diff
@@ -41,21 +41,21 @@
      */
     public static final int NOT_SPECIFIED = -1;
 
-    public static final IdentityProvider GOOGLE = new IdentityProvider(
-            "Google",
-            R.bool.google_enabled,
-            R.string.google_discovery_uri,
+    public static final IdentityProvider OPEN_ID = new IdentityProvider(
```

Comment:
I don't mind removing the word "Google" here for the demo, but I'm not sure about calling it "OpenID". Say an app has 3 providers, likely they are *all* OpenID providers.  So what about calling it "EXAMPLE_PROVIDER" or something like that?  I don't want people thinking they can only have 1 OpenID provider in the app, or that other provider types (e.g. SAML) are supported.